### DISCUSSION TITLE
flv.js 目前似乎缺少维护，是否考虑将其替换为 mpegts.js

### DISCUSSION THREAD

**Discussion by opoet** [spencerwooo/onedrive-vercel-index#418] [General]
- https://github.com/bilibili/flv.js/issues/724
- https://github.com/bilibili/flv.js/pull/727

> [mpegts.js](https://github.com/xqq/mpegts.js) 基于 [flv.js](https://github.com/bilibili/flv.js) 改造而来

Edit: 
- https://github.com/xqq/mpegts.js/issues/5
- https://github.com/xqq/mpegts.js/issues/39

**Comment by myl7** [comment]
之前选用 `flv.js` 是因为 `DPlayer` 的文档里只有 `flv.js` 的示例。
如果 `mpegts.js` 能够以相似的方式配置的话，当然是 `mpegts.js` 更佳。
但我不确定 `mpegts.js` 有没有相似的 API。

**Reply by opoet** [reply]
下面分别是两个库的 api 说明

https://github.com/bilibili/flv.js/blob/master/docs/api.md
https://github.com/xqq/mpegts.js/blob/master/docs/api.md

edit: 区别与不同 https://github.com/xqq/mpegts.js/issues/39

**Comment by spencerwooo** [comment]
可以考虑，但是有更多两个库之间区别的相关文档吗？

**Reply by opoet** [reply]
~~似乎没有~~，下面分别是两个库的 api 说明

https://github.com/bilibili/flv.js/blob/master/docs/api.md
https://github.com/xqq/mpegts.js/blob/master/docs/api.md

edit: 区别与不同 https://github.com/xqq/mpegts.js/issues/39
